pages/index.html

```diff
@@ -5,8 +5,23 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="../styles/style.css" rel="stylesheet">
 </head>
-<body>
-  
-</body>
```

Comment:
It looks like you deleted the body tag...  while a page will _technically_ render without one it does kind of weird things...